iphone/Maps/UI/PlacePage/Components/PlacePagePreviewViewController.swift

```diff
@@ -232,10 +232,19 @@ final class PlacePagePreviewViewController: UIViewController {
     attributedString.append(stateString);
     if (details != nil)
     {
-      let detailsString = NSAttributedString(string: " • " + details!,
-                                             attributes: [NSAttributedString.Key.font: UIFont.regular14(),
-                                                          NSAttributedString.Key.foregroundColor: UIColor.blackSecondaryText()]);
-      attributedString.append(detailsString);
+      if (placePagePreviewData.schedule.state == .closed){
+        let detailsString = NSAttributedString(string: "" + details!,
+                                               attributes: [NSAttributedString.Key.font: UIFont.regular14(),
+                                                            NSAttributedString.Key.foregroundColor: UIColor.systemRed]);
+        attributedString.append(detailsString);
```

Comment:
Why is this line under the "if" clause?

Reply:
> Why is this line under the "if" clause?

case .allDay:
setScheduleLabel(state: L("twentyfour_seven"),
stateColor: UIColor.systemGreen,
details: nil);

here the details are nil ,  so in the function setScheduleLabel there is no need to check the other lines that's why using an if clause is helpful